js/privatebin.js

```diff
@@ -2257,6 +2088,9 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
          */
         me.hasAttachment = function()
         {
+            if (!AttachmentViewer.$attachment.length) {
```

Comment:
If you are in `AttachmentViewer` you can just use `me` instead of referring to it with its full name.